.devops/pagopa-code-review-pipelines.yml

```diff
@@ -54,6 +54,7 @@ stages:
             api_wallet_basepath: '/webview-payment-wallet/v1'
             api_pm_basepath: '/pp-restapi-CD'
             api_timeout:  10000
+            npg_sdk_url:  'https://stg-ta.nexigroup.com/monetaweb/resources/hfsdk.js'
```

Comment:
ok! but the same change should be inserted in the dev and uat stages in the deployment pipeline.
**pagopa-deploy-pipelines.yml**